src/pages/docs/creators/ray/quickstart.md

```diff
@@ -58,18 +58,24 @@ wget https://github.com/golemfactory/ray-on-golem/raw/main/examples/simple-task.
 # Execute the app locally by starting a local ray instance on your computer
 python3 simple-task.py
 ```
+This particular script shows information about the cluster it is being run on 
+and also visualizes the number of tasks run on different nodes.
 
-Now that you know that Ray is working, execute the app on the cluster. Observe how Ray on Golem cluster expands during the computation
+Once you ensure the app works, you can feed it to your Ray on the Golem cluster
 
 ```bash
-# Run some ray-based code (that knows *nothing** about Golem) - this will either:
-# A) Run only on one node (the head node), if the autoscaler decides there is no need for a worker node
-# B) Or create worker node(s) on the Golem Network. Worker nodes will be later auto-terminated by the autoscaler)
-
 # Submit the app to be executed on your cluster
 ray submit golem-cluster.yaml simple-task.py 
 ```
 
+You can see the information about the cluster both before and after running the computations.
+Observe how, at first, the cluster consists of only one node, and how the autoscaler expands it, as the work progresses.
+
+
+You might need to submit the code more than once.
```

Comment:
@mateuszsrebrny as discussed, it could be better to provide a parameter to the script that differentiates the duration of the run, and run the script with a short duration the first time around and only then suggest the user to run it with a longer duration so that it causes autoscaling